garrysmod/lua/includes/util/color.lua

```diff
@@ -46,6 +46,86 @@ function COLOR:__tostring()
 
 end
 
+--[[---------------------------------------------------------
+	Returns unary color
+-----------------------------------------------------------]]
+function COLOR:__unm()

```

Comment:
I would expect this function to just negate all color components so that these two lines produce the same result
```lua
local invert1 = color_white - color
local invert2 = color_white + (-color)
````
The current implementation should rather be named `GetInverted` to be consistent with `Normalize`/`GetNormalized`.